### PR TITLE
First test: Answering Philippe's comments

### PR DESCRIPTION
**Pull request by EmmaRenauld**
Philippe tested the scripts. Here are modifs based on his feed-backs.

@ppoulin91

### CODE REVIEW

File: dwi_ml/data/hdf5_creation.py [hunk 1]
```diff
@@ -570,27 +570,30 @@ def _process_one_streamline_group(
...
             else:
-                bundles2 = [tmp_bundle_name]
+                bundles_sublist = [tmp_bundle_name]
 
             # Either a loop on "ALL" or a loop on only one file,
             # tmp_bundle_name.
-            for bundle_name in bundles2:
+            for bundle_name in bundles_sublist:
                 bundle_name = bundle_name.replace('*', subj_id)
                 bundle_file = subj_dir.joinpath(bundle_name)
```
Comment (ppoulin91) [inline dwi_ml/data/hdf5_creation.py:583]: There is still the problem that `bundle_name` is actually the bundle's path, even when not using `ALL`

Comment (EmmaRenauld) [reply dwi_ml/data/hdf5_creation.py:583]: oh, I just tested the 'all' case.

Comment (EmmaRenauld) [reply dwi_ml/data/hdf5_creation.py:583]: I'm not sure I see the issue. In the 'all' case, I fixed it with at the new line 514. In the other case, tmp_bundle_name is directly read from the config file, it shouldn't be the whole path.

**Review by ppoulin91 — APPROVED** [review]
Testé, ça fonctionne bine de mon côté

**Pull request merged**